frontend/packages/console-shared/src/hooks/useActivePerspective.ts

```diff
@@ -5,6 +5,11 @@ import {
   PerspectiveType,
 } from '@console/app/src/components/detect-perspective/perspective-context';
 
+const PERSPECTIVE_VISITED_SUFFIX = '_VISITED';
+
+export const getPerspectiveVisitedKey = (perspective: PerspectiveType): string =>
+  `${perspective}${PERSPECTIVE_VISITED_SUFFIX}`;
```

Comment:
User settings keys should use reverse domain notation with the `console` prefix
```suggestion
export const getPerspectiveVisitedKey = (perspective: PerspectiveType): string =>
  `console.perspective.visited.${perspective}`;
```